src/utils/date/index.js

```diff
@@ -0,0 +1,10 @@
+const formatDate = dateString => {
+  if(isValidDate(dateString))
+    return new Date(dateString + ' UTC').toLocaleString()
```

Comment:
The date is being modified after its check for validity. What if the act of modification causes the date to be invalid?

There should be a contract between the backend and frontend on the date format, but if that contract is inadvertently broken then I think we would like isValidDate to return false if appending UTC to the date causes it to be invalid.

Reply:
I agree with you Nick, we might have a problem if the contract is broken for some reason.
One suggestion it to change how we expose the dates through the API. I looked on the Manager list http call and noticed that the date is being transformed on the backend. 
![image](https://user-images.githubusercontent.com/22650820/144855105-bd85a963-822e-433e-8f66-c7e9a4754619.png)
![image](https://user-images.githubusercontent.com/22650820/144855136-1a62b99f-b8d5-4d6e-afa6-55c65d2bbf62.png)

![image](https://user-images.githubusercontent.com/22650820/144854906-0cbbe241-fff7-424f-b997-cf1f9e386cf9.png)

I think we just need to send the date in ISO format and leave the front end handle it.
The only contract we will follow is that the backend sends a UTC date.

What do you think?

Reply:
> I think we just need to send the date in ISO format and leave the front end handle it.
> The only contract we will follow is that the backend sends a UTC date.
> 
> What do you think?

Yes, I've been wanting to do that for a long time.

We're going to be combining the frontend and backend into one app/repository. See PR #769

It doesn't do any date transformations like the old backend. And sends all dates in ISO format.

I haven't implemented anything that formats the dates on the frontend, so if someone is wanted to they could implement that.